util/src/test/java/care/smith/fts/util/GICSConsentedPatientExtractorTest.java

```diff
@@ -83,7 +86,7 @@ void extractConsentedPatientWithUnknownPoliciesYieldsEmptyResult() {
   @Test
   void hasAllPolicies() {
     var result =
-        ConsentedPatientExtractor.hasAllPolicies(POLICY_SYSTEM, bundle1, POLICIES_TO_CHECK);
+        GICSConsentedPatientExtractor.hasAllPolicies(POLICY_SYSTEM, bundle1, POLICIES_TO_CHECK);
 
     assertThat(result).isTrue();
   }
```

Comment:
Am I missing something or should we add a test, that checks if the patient extractor inserted the given identifier system, and doesn't use the one from gICS ?